content/chainguard/chainguard-images/working-with-images/security-advisories/how-chainguard-issues/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "How Chainguard Issues Security Advisories"
+linktitle: "Chainguard's Advisories"
+aliases: 
+type: "article"
+description: "The life cycle of Chainguard-Issued Security Advisories"
+date: 2024-07-26T18:09:12+00:00
+lastmod: 2024-07-30T14:59:20+00:00
+draft: false
+tags: ["Overview", "Product", "Chainguard Images", "CVE"]
+images: []
+menu:
+  docs:
+    parent: "security-advisories"
+weight: 020
+toc: true
+---
+
+When you scan a newly-built Chainguard Image with a vulnerability scanner, typically, no CVEs will be reported. However, as software packages age, more vulnerabilities are reported, and CVEs begin to accumulate in images. When this happens, Chainguard releases security advisories to communicate these vulnerabilities to downstream Images users.
+
+Chainguard publishes its security advisories to a dedicated [Security Advisories page](https://images.chainguard.dev/security/) on its Images Registry. Here, you can find a complete listing of CVEs found in various Chainguard Images, including their CVE ID, affected packages, and vulnerability status. Essentially, these security advisories act as metadata for a security vulnerability.
+
+![Snapshot of the Chainguard Security Advisories Page](advisories-page.png)
+
+
+You can also find consumable Alpine-style `secdb` security advisory feeds at the following URLs:
+[Wolfi OS repository feed](https://github.com/wolfi-dev/os): https://packages.wolfi.dev/os/security.json
```

Comment:
```suggestion
* [Wolfi OS repository](https://github.com/wolfi-dev/os) feed: [packages.wolfi.dev/os/security.json](https://packages.wolfi.dev/os/security.json)
```
This becomes a muddled paragraph without some fancy formatting, I think this would look better but your call

Reply:
I think I lost the formatting from the Google Doc when I copied it over... thanks for pointing this out!